config/environments/development.rb

```diff
@@ -58,5 +58,12 @@
   # Allow disabling the webpack dev server from the settings
   config.webpack.dev_server.enabled = SETTINGS.fetch(:webpack_dev_server, true)
   config.webpack.dev_server.https = SETTINGS.fetch(:webpack_dev_server_https, false)
+
   config.hosts << SETTINGS[:fqdn]
+  additional_hosts = SETTINGS[:additional_hosts]
+  if additional_hosts.is_a?(Array)
+    additional_hosts.each do |host|
+      config.hosts << host
+    end
+  end
```

Comment:
I'm leaning to simply naming this `hosts`. You can replace a special value `FQDN` to add the FQDN setting. That way the logic becomes:
```suggestion
  additional_hosts = SETTINGS[:additional_hosts]
  if additional_hosts.is_a?(Array)
    additional_hosts.each do |host|
      config.hosts << host == 'FQDN' ? SETTINGS[:fqdn] : host
    end
  else
    config.hosts << SETTINGS[:fqdn]
  end
```

Then as a next step I'd simplify it and ensure it's always an array with `FQDN` as the single value by default (which you can do in `config/settings.rb`.

Now it does become confusing to have a config entry in settings.yaml documented, but only applicable to development. So I'd make sure it also applies to production environments too. This probably requires a change in the installer to properly work, but that's ok.

Reply:
you want SETTINGS[:additional_hosts] to be renamed SETTINGS[:hosts], right? I'm not understanding how config/settings.rb comes into play.

Reply:
> you want SETTINGS[:additional_hosts] to be renamed SETTINGS[:hosts], right?

Correct

> I'm not understanding how config/settings.rb comes into play.

How else will you deal with the value missing from `settings.yaml`? You don't want to duplicate that handling over multiple places.